src/Microsoft.OData.Core/RawValueWriter.cs

```diff
@@ -166,24 +152,6 @@ internal void Flush()
             }
         }
 
-        /// <summary>

```

Comment:
Reason for dropping `Start`/`StartAsync` and `End`/`EndAsync`? I thought they existed so that if the future we want to plug in some before before and after calling `WriteRawValue`/`WriteRawValueAsync` we would have a means to do so? We didn't even mark the methods as obsolete in ODL 8